### ISSUE TITLE
Getting error while generating files

### ISSUE THREAD

**Issue by technicalbirdVayuz** [frozeman/meteor-build-client#71]
![image](https://user-images.githubusercontent.com/18699624/174775209-01cd1d76-30df-4e3b-8de6-8e0b6385d30f.png)
Please check

Files are getting generated but does not contains any script tags, only basic html

**Comment by dr-dimitru** [comment]
Hello @technicalbirdVayuz ,

Make sure you have `meteor-bundled-css` and `meteor-bundled-js` in your template, example:

```html
<head>
  <!-- your header stuff... -->
  <!-- then typically add the css link at the bottom of the head -->
  <meteor-bundled-css />
</head>

<body>
  <!-- typically want to load all the js code at the top -->
  <meteor-bundled-js />
  <!-- The rest of your body stuff... -->
</body>
```

See [in-code annotation](https://github.com/frozeman/meteor-build-client/blob/master/meteor.js#L151) for in-depth details


Feel free to close it in case if the issue is solved on your end.

**Comment by dr-dimitru** [comment]
@technicalbirdVayuz also looking at your error, seems like you aren't on the latest `meteor-build-client` release. Please upgrade to the latest [`1.2.1`](https://github.com/frozeman/meteor-build-client/releases/tag/1.2.1) with `npm -g update` id necessary

**Comment by technicalbirdVayuz** [comment]
![image](https://user-images.githubusercontent.com/18699624/174815488-76124ed8-1d0f-4ba2-948c-30347c33c898.png)
Still not working

**Comment by dr-dimitru** [comment]
@technicalbirdVayuz whichever contains `<head>` and `<html>` elements

**Comment by dr-dimitru** [comment]
@technicalbirdVayuz I'm glad issue is solved for you. Let's keep this open, I'll work on suppressing error output

**Comment by dr-dimitru** [comment]
@technicalbirdVayuz in order to test this usa case can you post your `.meteor/packages` and `packages.json`? Can not figure out how to build an app without legacy client code

**Comment by dr-dimitru** [comment]
Closed by [95c3040](https://github.com/frozeman/meteor-build-client/commit/95c3040b9513723446f504d1a96b8d3228f0dfac)
